wasmcloud-chat/ops/broker-message-request.json

```diff
@@ -0,0 +1 @@
+{ "source_app_id": "website", "source_app_version": "1.0", "message_text": "Hello from the other side", "origin_user_id": "bob", "target_url": "wcchat://rooms/general", "created_on": 0, "access_token": "" }
```

Comment:
I believe `origin_user_id` should be `adele` here.

Reply:
Indeed. :monocle_face: